src/fragments/lib/ssr/js/getting-started.mdx

```diff
@@ -65,8 +71,6 @@ export async function getServerSideProps({ req }) {
 }
 ```
 
-Server-side functions that _don't_ have a `req` object (e.g. Next.js' `getStaticProps` & `getStaticPaths`) should still use `withSSRContext()`.
```

Comment:
I don't think we should remove this, please see [this comment](https://github.com/aws-amplify/docs/pull/4517/files#r941661690). Happy to chat about it more offline